config/application.rb

```diff
@@ -30,6 +30,7 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
     config.autoloader = :zeitwerk
+    config.i18n.default_locale = :'en-GB'
```

Comment:
Is this so we can print numbers in the right currency? Isn't it a bit of an overkill?

Reply:
Yes it is. Seems reasonable to me, this is a GB app. Without this we'd have to pass the locale to every call to the currency formatter.... (and I couldn't actually make that work for some reason I didn't get to the bottom of)